post_assets.py

```diff
@@ -0,0 +1,75 @@
+import sys
+import os
+import csv
+import django
+
+project_dir = os.path.dirname(os.path.abspath(__file__))
+sys.path.append(project_dir)
+django.setup()
+from core.models.asset import Asset, AssetModelNumber   # noqa
+
+
+def bulk_create_asset():
+    with open('assets.csv', 'r', ) as f:
+        skipped = dict()
+        skipped_records = []
+        inserted_records = []
+        data = csv.DictReader(f, delimiter='|')
+        for row in data:
+            model_number = AssetModelNumber.objects.\
+                filter(model_number=row['model_number'])\
+                .exists()
+            asset_code = Asset.objects.filter(asset_code=row['asset_code'])\
+                .exists()
+            serial_number = Asset.objects.filter(
+                serial_number=row['serial_number']).exists()
```

Comment:
we can have some logging here as well.. either info about the current row or some sort of progress indicator

Reply:
print statements ie